### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by gemy6315** [community/community#103108] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

``on: push
name: Build and Publish
jobs:
  shellLint:
    name: Shell Lint
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - name: Shell Lint
      uses: actions/bin/shellcheck@master
      with:
        args: entrypoint.sh
    - name: Build Docker
      uses: actions/docker/cli@master
      with:
        args: build -t npm .
    - name: Build
      uses: actions/npm@master
      with:
        args: install
    - name: Test
      uses: actions/npm@master
      with:
        args: test
    - name: Publish Filter
      uses: actions/bin/filter@master
      with:
        args: branch master
    - name: Publish
      uses: mikeal/merge-release@master
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}

**Comment by ghostinhershell** [comment]
Hi @gemy6315,

Hi @gemy6315, we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction. Otherwise, this post will be removed within 24-48 hours to keep our community nice and tidy 🧹.

Furthermore, I've removed your private information from this post (the email address you posted as the title) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @gemy6315! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by gemy6315** [reply]
> Hey there!  👋
> 
> Thanks for posting in the GitHub Community, @gemy6315! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
> 
> I've gone ahead and moved this to the correct category for you. Good luck!
